packages/@remirror/core-utils/src/command-utils.ts

```diff
@@ -170,13 +173,20 @@ export function setBlockType(
 
       if (node.type === type) {
         applicable = true;
+        activeAttrs = node.attrs;
```

Comment:
Hey @ifiokjr – giving this functionality a try in v60 and it doesn't seem to be working as intended – my guess is this `if` statement, where it's checking that the node type === type before considering it's attributes, but by definition the `nodeType` is only the same in one toggle direction.